pkg/httpapi/api.go

```diff
@@ -161,6 +166,34 @@ func (a *APIRouter) GetApplication(w http.ResponseWriter, r *http.Request) {
 	marshalResponse(w, appEnvironments)
 }
 
+func (a *APIRouter) ListApplications(w http.ResponseWriter, r *http.Request) {
+	user := r.URL.Query().Get("user")
```

Comment:
Will this work if the user is using some other language other than English?  How about a non-English user name?